### DISCUSSION TITLE
about lenses

### DISCUSSION THREAD

**Discussion by etteei** [BrianHanke/lentil-bhgc#5] [Q&A] [answered]
Hello, I'd like to know how many lenses does Lentil-BHGC have? Can I create lenses by myself?

**Comment by BrianHanke** [comment, marked as answer]
Lentil-BHGC includes all the original Lentil lens models, listed here: https://github.com/BrianHanke/lentil-bhgc/blob/main/polynomial-optics/database/lenses.json

If you can provide data in the format Lentil needs, then yes, you can add new lenses. The catch is that they have to be compiled into a new Lentil build.

**Reply by etteei** [reply]
Thanks for your reply. I already bought the plugin yesterday.
